tests/integration/asserts/time.py

```diff
@@ -0,0 +1,35 @@
+from datetime import timedelta, datetime, timezone
+
+
+class _WithinBounds:
+
+    def __init__(self, lower_bound, upper_bound):
+        self._lower_bound = lower_bound
+        self._upper_bound = upper_bound
+
+    def __eq__(self, other):
+        assert isinstance(other, int)
+        return self._lower_bound <= other <= self._upper_bound
+
+    def __str__(self):
+        return f"{self._lower_bound} <= x <= {self._upper_bound}"
+
+
+def time_after(lower_bound):
```

Comment:
This doesn't assert after, but after until now. I think making the upper bound on `time_within` optional and defaulting to now is fine.

Reply:
Yeah, I was indeed thinking about a better name.